### DISCUSSION TITLE
Builds Overview Page > List Items > Artifacts and Build Number missing

### DISCUSSION THREAD

**Discussion by adamdansky** [codemagic-ci-cd/codemagic-docs#1350] [Q&A] [answered]
Starting yesterday, Artifacts and Version / Build Number are no longer present on the list items on the Builds Overview page. 

- no recent changes to our codemagic config
- unsure if correlated, but started after a build timed out

Before, with artifacts
![list-item-with-artifact](https://user-images.githubusercontent.com/1322667/214917239-f78289ad-7303-46fb-ae69-d11796cdae5f.png)

Now, without artifacts
![list-item-without-artifact](https://user-images.githubusercontent.com/1322667/214917374-0892bc5c-f71a-4dad-8050-77dd8035a442.png)

Thanks for any info or tips on how to remedy. 

And also curious if this may be indicative of other less obvious build issues?

**Comment by adamdansky** [comment]
Artifacts and version / build number have returned to our list items for new builds.

**Comment by himesh-cm** [comment, marked as answer]
Hi @adamdansky, Possible that it was an intermittent issue. Let us know in case you have any queries.
